mrblib/specinfra/command/gentoo/base/package.rb

```diff
@@ -1,7 +1,24 @@
 class Specinfra::Command::Gentoo::Base::Package < Specinfra::Command::Linux::Base::Package
   class << self
     def check_is_installed(package, version=nil)
-      "eix #{escape(package)} --installed | grep -v \"No matches found.\""
+      "eix #{escape(package)} --installed | grep -v \"No matches found\""
+    end
+
+    def get_version(package, opts=nil)
+      "equery -q list #{package} | sed -e 's!^.*/?#{package}-!!'"
+    end
+
+    def install(package, version=nil, option='')
+      if version
+        full_package = "=#{package}-#{version}"
+      else
+        full_package = package
```

Comment:
FYI, this is called _atom._ Read the man page emerge(1) and ebuild(5) for details